front/admin/db.ts

```diff
@@ -77,15 +81,19 @@ async function main() {
   await Plan.sync({ alter: true });
   await Subscription.sync({ alter: true });
 
+  await AgentGenerationConfiguration.sync({ alter: true });
+
+  await AgentRetrievalConfiguration.sync({ alter: true });
   await AgentDustAppRunConfiguration.sync({ alter: true });
   await AgentDustAppRunAction.sync({ alter: true });
   await AgentTablesQueryConfiguration.sync({ alter: true });
   await AgentTablesQueryConfigurationTable.sync({ alter: true });
   await AgentTablesQueryAction.sync({ alter: true });
+  await AgentProcessConfiguration.sync({ alter: true });
+  await AgentProcessAction.sync({ alter: true });
 
-  await AgentGenerationConfiguration.sync({ alter: true });
-  await AgentRetrievalConfiguration.sync({ alter: true });
   await AgentDataSourceConfiguration.sync({ alter: true });
+
```

Comment:
Since we are here, should we move `AgentConfiguration` so it's before the actions ? I don't see how this can work from scratch otherwise since the actions reference the agent config.

Reply:
yes!